war/src/main/java/io/lumeer/engine/controller/DatabaseInitializer.java

```diff
@@ -59,6 +59,12 @@ public void init() {
                .append(LumeerConst.Project.UserRoles.ATTR_PROJECT_ID, LumeerConst.Index.ASCENDING)
                .append(LumeerConst.Project.UserRoles.ATTR_USER_ROLE, LumeerConst.Index.ASCENDING), true);
       }
+
+      if (!dataStorage.hasCollection(LumeerConst.Organization.COLLECTION_NAME)) {
+         dataStorage.createCollection(LumeerConst.Organization.COLLECTION_NAME);
+         dataStorage.createIndex(LumeerConst.Organization.COLLECTION_NAME, new DataDocument(LumeerConst.Organization.ATTR_ORG_ID, LumeerConst.Index.ASCENDING)
+               .append(dataStorageDialect.concatFields(LumeerConst.Organization.ATTR_USERS, LumeerConst.Organization.ATTR_USERS_USERNAME), LumeerConst.Index.ASCENDING), false);
+      }
```

Comment:
@kubedo8 Thanks. Except of creating separated indexes for `id`, `users` and `name` I should let this code there?
```
dataStorage.createIndex(LumeerConst.Organization.COLLECTION_NAME, new DataDocument(LumeerConst.Organization.ATTR_ORG_ID, LumeerConst.Index.ASCENDING)
               .append(dataStorageDialect.concatFields(LumeerConst.Organization.ATTR_USERS, LumeerConst.Organization.ATTR_USERS_USERNAME), LumeerConst.Index.ASCENDING), false);
```